README.md

```
@@ -122,12 +122,12 @@ Promptfoo commands are wrapped in Makefile targets and read environment from the
 - Start a continuous evaluation and write results to `promptfoo/output.yml`:
 
     ```shell
-    make propmtfoo
+    make pf-e2e
     ``` 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)